tools/wptrunner/wptrunner/browsers/chrome_android.py

```diff
@@ -86,15 +88,18 @@ def __init__(self, logger, binary, webdriver_binary="chromedriver",
         self.server = ChromeDriverServer(self.logger,
                                          binary=webdriver_binary,
                                          args=webdriver_args)
+        self.setup_adb_reverse()
 
     def _adb_run(self, args):
         self.logger.info('adb ' + ' '.join(args))
         subprocess.check_call(['adb'] + args)
 
-    def setup(self):
+    def setup_adb_reverse(self):
         self._adb_run(['wait-for-device'])
         self._adb_run(['forward', '--remove-all'])
         self._adb_run(['reverse', '--remove-all'])
+        # "adb reverse" basically forwards network connection from device to
```

Comment:
Nit:

```suggestion
        # "adb reverse" forwards network connection from device to host.
```

And drop the next line.

Reply:
Done.